WORKSPACE

```diff
@@ -16,30 +16,72 @@ load("@aspect_bazel_lib//lib:repositories.bzl", "aspect_bazel_lib_dependencies")
 
 aspect_bazel_lib_dependencies()
 
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "sysroot",
+    build_file_content = """\
+filegroup(
+    name = "sysroot",
+    srcs = glob(["**"]),
+    visibility = ["//visibility:public"],
+)
+""",
+    sha256 = "84656a6df544ecef62169cfe3ab6e41bb4346a62d3ba2a045dc5a0a2ecea94a3",
+    urls = ["https://commondatastorage.googleapis.com/chrome-linux-sysroot/toolchain/2202c161310ffde63729f29d27fe7bb24a0bc540/debian_stretch_amd64_sysroot.tar.xz"],
+)
+
 load("//toolchain:defs.bzl", "gcc_register_toolchain")
 
-flags = [
+common_flags = [
     "-Wall",
     "-Wextra",
     "-fdiagnostics-color=always",
 ]
 
+ldflags = common_flags + [
+    "-B{sysroot}/usr/lib/x86_64-linux-gnu",
```

Comment:
nit: who interpolates `{sysroot}` and are other variables available as well? a bit of docs would be nice

Reply:
The doc already exists in the attribute.